_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+---
+layout: post
+title:  "Voicemeeter, an introduction to mixing audio"
+author: flavius
+categories: [ audio ]
+image: ""
+featured: false
+---
+Want to try mixing your audio and eventually have an easier time making adjustments? Don't want to invest in a physical mixer or GoXLR/Elgato Wave with other Digital Mixing software? Come check out this introduction to Voicemeeter, a digital mixing software to help organize your audio.
+
+### - Introduction
+
+Greetings fellow streamers and content creators. Are you interested in taking your audio to the next level but don't have a physical mixer or GoXLR type device such as the GoXLR, GoXLR Mini or Elgato Wave software included with the Elgato Wave microphones? Maybe you are interested in seeing how audio mixing works but don't want to fully commit to a physical device or put tons of money into just to try out? Well, guess what? There are options to try out! While it might not exactly stack up to having physical knobs and sliders to adjust, there is a digital option called Voicemeeter.
+
+You might ask, hey Flav, what is voice meeter? You know what? I can answer that! Voicemeeter is a digital mixing software available as donationware for windows that allows you to pull in different sources of audio and allow for advanced audio routing and configuration. The base version of voicemeeter and banana version are free. The potato version has 30 days and then it periodically asks you for a one time donation to get a key and unlock it fully.
+
+Before we get too far into this, I want to state my experience for reference. I have personally used all three versions of voicemeeter. I swapped between Banana and Potato frequently for over a year and a half as of Jan 2021. I did a great many things with it and I absolutely love the software. Am I still using it? My personal setup doesn't really require the use of it anymore so not entirely. I still fire up both Banana and Potato versions periodically to refresh my knowledge on it so I can help others fix any problems or set it up. If at any point you have questions about Voicemeeter or even other subjects, the Library by Streamsounds does have a discord server ready for you to hop into if you need any help or just want to chat. Technology interests us greatly and we would love to chat! Anyways, back to the article!
```

Comment:
1. You did "a great many things with it", but what did you do? Can you give some specifics?
2. Don't ask the question of whether you are still using it or not. Say something like, "Although I've moved on to using a physical mixer, Voicemeeter taught me all the basics of handling multiple audio sources so I could hit the ground running with a physical mixer."
3. In the sentence that starts with "If at any point you have questions...", give a stronger call-to-action like "join the [StreamSounds Discord Server]({{ site.streamsounds.discord}}) to get the help you need with setting up Voicemeeter." This makes it more direct on what they should do for help.

Reply:
1. I removed the great many things with it, posing that in the article represents I might go into more detail and really it isn't necessary for the article and setup. So I'll save it for writing a different article down the road. 
2. I think this is removed now from the new revision.
3. Take a look at the new call to action sort of things I did in Voicemeter banana article. Maybe some feedback on that one?